assignment1.txt

```diff
@@ -0,0 +1,17 @@
+1. A compiler is a program that takes code written in C and translates it into Assembly, a compiler is used so a computer can be one step closer to reading code
+
+
+
+2.
+
+a: compiler error-it is syntaxual they forgot the ";"
+
+b: compiler error - again sytax error - you cant begin with a non alphabetic character
+
+
+c: runtime error - nothing is wrong here technically but the computer cannot divide by 0
+
+
+Compiler errors are human caused while runtime errors are the limitations of the computer
```

Comment:
sort of - compile time errors are syntax errors/errors that prevent a computer from properly compiling. Run time errors occur during run time and have more to do with the logic of your program.